Dockerfile

```diff
@@ -0,0 +1,11 @@
+# syntax=docker/dockerfile:1
+FROM ruby:3.1.3
+WORKDIR /crimethinc
+COPY . .
+RUN apt-get update -qq && apt-get install -y nodejs postgresql-client ruby-full git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison build-essential libyaml-dev libreadline-dev libncurses5-dev libffi-dev libgdbm-dev
+
+RUN gem install bundler --conservative && gem install os && bundle install
+
+CMD bundle exec rails db:create && bundle exec rails db:migrate && ./script/server
```

Comment:
You probably don't want to run `db:create` each time you start the application. You should probably provide an entrypoint that point to `bin/rails` and set the command to `bin/rails server`. Since the entrypoint is gonna be `bin/rails`, we're gonna be able to do stuff like `docker run crimethinc:latest db:create db:migrate`.

Also, you can run multiple task as once, so you could do `bundle exec db:create db:migrate` in one command.

Reply:
This was great advice. It ended up being so cleaner and more usable.